src/main/java/com/sequenceiq/cloudbreak/domain/InstanceStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.sequenceiq.cloudbreak.domain;
+
+public enum InstanceStatus {
+    DECOMMISIONED, ALIVE, UNREGISTERED, TERMINATED
```

Comment:
instead of `ALIVE` we should name this status `REGISTERED` or `ASSIGNED`, because decommissioned and unregistered instances are also alive.